lib/Parser/StandardChars.h

```diff
@@ -225,6 +225,8 @@ namespace UnifiedRegex
             return true;
         }
 
+        TrivialCaseMapper() {}
+        
```

Comment:
nit pick: trailing spaces. What makes this constructor needed... the CompileAssert?